frontend/src/metabase/query_builder/components/filters/pickers/DatePicker/DatePicker.tsx

```diff
@@ -180,7 +182,7 @@ const DatePicker: React.FC<Props> = props => {
     <div className={cx(className)} data-testid="date-picker">
       {!operator || showShortcuts ? (
         <DatePickerShortcuts
-          className="p2"
+          className={withPadding ? "p2" : ""}
```

Comment:
If there's a better way to do this without messing up the padding in other widgets, I'm open to suggestions